src/index.ts

```diff
@@ -220,20 +220,22 @@ class ServerlessSSMPublish {
         Name: param.path,
         Description: param.description || `Placed by ${this.serverless.service.getServiceName()} - serverless-ssm-plugin`,
         Value: typeof param.value === 'string' ? param.value : yaml.safeDump(param.value),
-        Overwrite: true,
-        Type: param.secure ? 'SecureString' : 'String',
+        Type: param.type ? param.type :  param.secure ? 'SecureString' : 'String',
```

Comment:
Remove extraneous space please.

```suggestion
        Type: param.type ? param.type : param.secure ? 'SecureString' : 'String',
```